hypothesisfuzzer/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .fuzzer import Fuzzer # noqa
+name = "hypothesisfuzzer"
```

Comment:
Why is this here? Is this standard practice?

Reply:
Seems to be standard practice...